testing/tests/DevExpress.ui.widgets.editors/dateRangeBox.tests.js

```diff
@@ -3893,3 +3894,164 @@ QUnit.module('calendarOptions', moduleConfig, () => {
         });
     });
 });
+
+QUnit.module('Aria accessibility', moduleConfig, () => {
+    QUnit.test('aria-owns attribute on root element', function(assert) {
+        assert.strictEqual(this.$element.attr('aria-owns'), undefined, 'aria-owns');
+        assert.strictEqual($(this.instance.startDateField()).attr('aria-owns'), undefined, 'aria-owns on startDateInput');
+        assert.strictEqual($(this.instance.endDateField()).attr('aria-owns'), undefined, 'aria-owns on endDateInput');
+
+        this.instance.open();
+
+        assert.strictEqual(this.$element.attr('aria-owns'), $(this.instance.content()).attr('id'), 'aria-owns');
+        assert.strictEqual($(this.instance.startDateField()).attr('aria-owns'), undefined, 'aria-owns on startDateInput');
+        assert.strictEqual($(this.instance.endDateField()).attr('aria-owns'), undefined, 'aria-owns on endDateInput');
+
+        this.instance.close();
+
+        assert.strictEqual(this.$element.attr('aria-owns'), undefined, 'aria-owns');
+        assert.strictEqual($(this.instance.startDateField()).attr('aria-owns'), undefined, 'aria-owns on startDateInput');
+        assert.strictEqual($(this.instance.endDateField()).attr('aria-owns'), undefined, 'aria-owns on endDateInput');
+    });
+
+    QUnit.test('aria-expanded attributes on inputs', function(assert) {
+        assert.strictEqual($(this.instance.startDateField()).attr('aria-expanded'), 'false', 'aria-expanded on startDateInput');
+        assert.strictEqual($(this.instance.endDateField()).attr('aria-expanded'), 'false', 'aria-expanded on endDateInput');
+
+        this.instance.open();
+
+        assert.strictEqual($(this.instance.endDateField()).attr('aria-expanded'), 'true', 'aria-expanded on startDateInput');
+        assert.strictEqual($(this.instance.endDateField()).attr('aria-expanded'), 'true', 'aria-expanded on endDateInput');
+
+        this.instance.close();
+
+        assert.strictEqual($(this.instance.startDateField()).attr('aria-expanded'), 'false', 'aria-expanded on startDateInput');
+        assert.strictEqual($(this.instance.endDateField()).attr('aria-expanded'), 'false', 'aria-expanded on endDateInput');
+    });
+
+    QUnit.test('aria-expanded attributes on inputs if opened is true on initialization', function(assert) {
+        this.reinit({
+            opened: true
+        });
+
+        assert.strictEqual($(this.instance.endDateField()).attr('aria-expanded'), 'true', 'aria-expanded on startDateInput');
+        assert.strictEqual($(this.instance.endDateField()).attr('aria-expanded'), 'true', 'aria-expanded on endDateInput');
+
+        this.instance.close();
+
+        assert.strictEqual($(this.instance.startDateField()).attr('aria-expanded'), 'false', 'aria-expanded on startDateInput');
```

Comment:
- it's already checked in test above
- it does not meet the name

Reply:
https://github.com/DevExpress/DevExtreme/pull/24798/commits/fadda309c6c15b8d9459ef9e5c474605c0ef1a54